lib/auth/grpcserver.go

```diff
@@ -230,6 +230,10 @@ func (g *GRPCServer) CreateAuditStream(stream proto.AuthService_CreateAuditStrea
 			}
 			eventStream, err = auth.CreateAuditStream(stream.Context(), session.ID(create.SessionID))
 			if err != nil {
+				// Log the reason why audit stream creation failed. This will
+				// surface things like AWS/GCP/MinIO credential/configuration
+				// errors.
+				log.Errorf("Failed to create audit stream: %q.", err)
```

Comment:
`GRPCServer` embeds a log entry.
```suggestion
				g.Errorf("Failed to create audit stream: %q.", err)
```

Reply:
Fixed.